consensus/src/processes/ghostdag/protocol.rs

```diff
@@ -115,14 +118,19 @@ impl<T: GhostdagStoreReader, S: RelationsStoreReader, U: ReachabilityService, V:
 
         let blue_score = self.ghostdag_store.get_blue_score(selected_parent).unwrap() + new_block_data.mergeset_blues.len() as u64;
 
-        let added_blue_work: BlueWorkType = new_block_data
-            .mergeset_blues
-            .iter()
-            .cloned()
-            .map(|hash| if hash.is_origin() { 0.into() } else { calc_work(self.headers_store.get_bits(hash).unwrap()) })
-            .sum();
+        let blue_work: Uint192 = if self.use_score_as_work {
```

Comment:
In what cases does it make sense to use score as work? And why?

Reply:
Based on usage, this is done for levels higher than level 0. blue score as blue work when level != 0

Reply:
On higher levels it makes more sense to just use blue score as this treats each block in the level's worth as 1